Extension/readme.developer.md

```diff
@@ -0,0 +1,338 @@
+# Developer Documentation
+
+## Table of Contents
+- [Developer Documentation](#developer-documentation)
```

Comment:
Can you remove `[Developer Documentation](#developer-documentation)` and move the stuff to the right? That will reduce the indentation. And can you remove the Table of Contents link too? e.g.

```markdown
- [Setup](#setup)
    - [Required Tools](#required-tools)
    - [Setting up the repository](#setting-up-the-repository)
- [Building the Extension](#building-the-extension)
    - [When using F5 Debug](#when-using-f5-debug)
    - [From Inside VSCode](#from-inside-vscode)
    - [From Command line](#from-command-line)
- [Use of an isolated VSCode environment](#use-of-an-isolated-vscode-environment)
- [Testing](#testing)
    - [Unit tests](#unit-tests)
    - [Scenario Tests](#scenario-tests)
- [Scripts](#scripts)
    - [`yarn scripts`](#yarn-scripts)
    - [`yarn show`](#yarn-show)
    - [`yarn clean`](#yarn-clean)
    - [`yarn test`](#yarn-test)
    - [`yarn code`](#yarn-code)
    - [`yarn generate-native-strings`](#yarn-generate-native-strings)
    - [`yarn generate-options-schema`](#yarn-generate-options-schema)
    - [`yarn copy-walkthrough-media`](#yarn-copy-walkthrough-media)
    - [`yarn prep`](#yarn-prep)
    - [`yarn lint`](#yarn-lint)
    - [`yarn compile`](#yarn-compile)
    - [`yarn watch`](#yarn-watch)
    - [`yarn verify`](#yarn-verify)
    - [`yarn webpack`](#yarn-webpack)
    - [`yarn postinstall`](#yarn-postinstall)
```

Reply:
I used an extension to generate the TOC.  ('markdown all in one') 

I added settings.json entries to change the indent and the generated TOC depth. 
I've also added a specific indent for markdown to be four spaces.